build.rs

```diff
@@ -8,12 +8,12 @@ fn main() {
     cfg_aliases! {
         // Systems.
         android_platform: { target_os = "android" },
-        wasm_platform: { target_family = "wasm" },
+        wasm_platform: { all(target_family = "wasm", not(target_os = "emscripten")) },
         macos_platform: { target_os = "macos" },
         ios_platform: { target_os = "ios" },
         windows_platform: { target_os = "windows" },
         apple: { any(target_os = "ios", target_os = "macos") },
-        free_unix: { all(unix, not(apple), not(android_platform)) },
+        free_unix: { all(unix, not(apple), not(android_platform), not(target_os = "emscripten")) },
```

Comment:
Could we just `not(wasm_platform)` here?

Reply:
Emscripten is now also excluded from `wasm_platform`, so that wouldn't be accounted for here.
(I'm not sure why Emscripten has `unix` to begin with anyway ...)

Reply:
Ah, yeah, it'll double negate.